bigtable/beam/bulk-data-generator/src/main/java/bigtable/BulkWrite.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bigtable;
+
+import com.google.bigtable.admin.v2.Cluster;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminClient;
+import com.google.cloud.bigtable.admin.v2.BigtableTableAdminSettings;
+import com.google.cloud.bigtable.admin.v2.models.CreateTableRequest;
+import com.google.cloud.bigtable.beam.CloudBigtableIO;
+import com.google.cloud.bigtable.beam.CloudBigtableTableConfiguration;
+import com.google.cloud.bigtable.grpc.BigtableClusterName;
+import com.google.cloud.bigtable.grpc.BigtableClusterUtilities;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.io.GenerateSequence;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.hadoop.hbase.client.Mutation;
+import org.apache.hadoop.hbase.client.Put;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.joda.time.Duration;
+
+public class BulkWrite {
+
+  static final double TB_PER_TABLE = .5;
+  static final long MB_PER_ROW = 1;
+  static final long ONE_MB = 1000 * 1000;
+  static final long ONE_GB = 1000 * ONE_MB;
+  static final long ONE_TB = 1000 * ONE_GB;
+  static final long MB_PER_SEC = 120;
+
+  static final String COLUMN_FAMILY = "cf";
```

Comment:
Is this a value that a developer/customer might want to customize, depending on their use case?

Reply:
Breaking the file won't really work since I need to run the code on dataflow during the test or it will run for a very long time.

Re: column family: This isn't really something developers are going to run since it just puts random data into Bigtable, it's more of a tool for demos. This also creates the tables so it will always have that family.